Makefile

```
@@ -178,3 +186,17 @@ installer-via-docker:
 	@echo "Building a cifuzz Linux installer"
 	mkdir -p build
 	DOCKER_BUILDKIT=1 docker build --platform linux/amd64 -f docker/cifuzz-builder/Dockerfile . --target bin --output build/bin
+
+.PHONY: docker/cifuzz-runner
+docker/cifuzz-runner: installer-via-docker
+	DOCKER_BUILDKIT=1 docker build -t cifuzz-runner -f docker/cifuzz-runner/Dockerfile .
```

Please consider adding this to support M1 Macs as well.
```suggestion
	DOCKER_BUILDKIT=1 docker build --platform linux/amd64 -t cifuzz-runner -f docker/cifuzz-runner/Dockerfile .
```

---

done